src/algebra/classical_lie_algebras.lean

```diff
@@ -0,0 +1,74 @@
+import algebra.lie_algebra
+import linear_algebra.matrix
+
```

Comment:
Please add a copyright header and module doc.

Reply:
Thanks for the reminded; now done, and apologies for omitting it at first.